Magento_Contact/templates/form.phtml

```diff
@@ -0,0 +1,89 @@
+                <?php /* @escapeNotVerified */ echo __('Name') ?>
+            </label>
+
+            <input type="text"
+                    name="name"
+                    id="name"
+                    class="input__field"
+                    title="<?php /* @escapeNotVerified */ echo __('Name') ?>"
+                    value="<?php echo $block->escapeHtml($this->helper('Magento\Contact\Helper\Data')->getPostValue('name') ?: $this->helper('Magento\Contact\Helper\Data')->getUserName()) ?>"
+                    data-validate="{required:true}"
+            />
+        </div>
+
+        <div class="input field required">
+            <label for="email" class="input__label">
+                <?php /* @escapeNotVerified */ echo __('Email') ?>
+            </label>
+
+            <input type="email"
+                    name="email"
+                    id="email"
+                    class="input__field"
+                    title="<?php /* @escapeNotVerified */ echo __('Email') ?>"
+                    value="<?php echo $block->escapeHtml($this->helper('Magento\Contact\Helper\Data')->getPostValue('email') ?: $this->helper('Magento\Contact\Helper\Data')->getUserEmail()) ?>"
+                    data-validate="{required:true, 'validate-email':true}"
+            />
+        </div>
+
+        <div class="input field">
+            <label for="telephone" class="input__label">
+                <?php /* @escapeNotVerified */ echo __('Phone Number') ?>
+            </label>
+
+            <input type="text"
+                    name="telephone"
+                    id="telephone"
+                    class="input__field"
+                    title="<?php /* @escapeNotVerified */ echo __('Phone Number') ?>"
+                    value="<?php echo $block->escapeHtml($this->helper('Magento\Contact\Helper\Data')->getPostValue('telephone')) ?>"
+            />
+        </div>
+
+        <div class="input comment required">
+            <label for="comment" class="input__label">
+                <?php /* @escapeNotVerified */ echo __('What’s on your mind?') ?>
+            </label>
+
+            <textarea class="input__field input__field--textarea"
+                        name="comment"
+                        id="comment"
+                        title="<?php /* @escapeNotVerified */ echo __('What’s on your mind?') ?>"
+                        cols="5"
+                        rows="3"
+                        data-validate="{required:true}"
+            ><?php echo $block->escapeHtml($this->helper('Magento\Contact\Helper\Data')->getPostValue('comment')) ?></textarea>
+        </div>
+
+        <?php echo $block->getChildHtml('form.additional.info'); ?>
+
+        <button class="button submit primary" 
+                title="<?php /* @escapeNotVerified */ echo __('Submit') ?>"
+                type="submit"
+        >
+            <?php /* @escapeNotVerified */ echo __('Submit') ?>
+        </button>
+    </form>
+</div>
```

Comment:
you can add some wrapper for the contact form b/c it looks bit strange when it has 100% content width. Maybe it can be thiner and centered?